doc/classes/VisualShaderNodeColorFunc.xml

```diff
@@ -1,21 +1,40 @@
 <?xml version="1.0" encoding="UTF-8" ?>
 <class name="VisualShaderNodeColorFunc" inherits="VisualShaderNode" category="Core" version="3.2">
 	<brief_description>
+		A [Color] function to be used within the visual shader graph.
 	</brief_description>
 	<description>
+		Accept a [Color] to the input port and transform it according to [member function].
 	</description>
 	<tutorials>
 	</tutorials>
 	<methods>
 	</methods>
 	<members>
 		<member name="function" type="int" setter="set_function" getter="get_function" enum="VisualShaderNodeColorFunc.Function" default="0">
+			A function to be applied to the input color. See [enum Function] for options.
 		</member>
 	</members>
 	<constants>
 		<constant name="FUNC_GRAYSCALE" value="0" enum="Function">
+			Converts the color to grayscale using the following formula:
+			[codeblock]
+			vec3 c = input;
+			float max1 = max(c.r, c.g);
+			float max2 = max(max1, c.b);
+			float max3 = max(max1, max2);
+			return vec3(max3, max3, max3);
+			[/codeblock]
 		</constant>
 		<constant name="FUNC_SEPIA" value="1" enum="Function">
+			Applies sepia tone effect using the following formula:
+			[codeblock]
+			vec3 c = input;
+			float r = (c.r * .393) + (c.g *.769) + (c.b * .189);
+			float g = (c.r * .349) + (c.g *.686) + (c.b * .168);
+			float b = (c.r * .272) + (c.g *.534) + (c.b * .131);
```

Comment:
I'd use explicit `0` for the decimals: `(c.r * 0.393) + (c.g * 0.769) + (c.b * 0.189);` (also there's a missing space after the second `*` on each line).